webapps/density-custom-chart/webapp.json

```diff
@@ -0,0 +1,39 @@
+{
+    "meta": {
+        "label": "Density map",
+        "description": "Generate a density map from geospatial data. Takes as input WKT points.",
```

Comment:
Well Known Text, according to previous development  we should say WKT since its pretty common, will keep this in mind if global change occurred on that convention